portfolio/src/main/webapp/script.js

```diff
@@ -52,41 +52,56 @@ function addRandomDestination() {
 }
 
 
+async function checkLoggedIn() {
+  const loginResponse = await fetch('/login', {method: 'POST'});
+  const loginJson = await loginResponse.json();
+  addLogInOutButton(loginJson.url, loginJson.isLoggedIn);
```

Comment:
Kinda weird that a function called `checkLoggedIn` changes the dom. 

Like if this is called multiple times is it going to repopulate the button?